### PR TITLE
Change vipps-checkout swagger links to point to the new website. 

### PR DESCRIPTION
**Pull request by rebekaburnett**
Also, fix a few heading levels to get rid of warnings

### CODE REVIEW

File: vipps-checkout-api.md [hunk 1]
```diff
@@ -201,7 +200,7 @@ with headers
 
 The last four headers (starting with `Vipps-System-`) are meant to identify your system (and plugin). Please use self-explanatory, human readable and reasonably short values.
 
-All fields of the request body are described in our [Swagger](https://vippsas.github.io/vipps-checkout-api/#/Session/post_v2_session). The dynamic logistics callback is found by clicking the "Callback" tab in the Swagger.
+All fields of the request body are described in our [Swagger](https://vippsas.github.io/vipps-developer-docs/api/checkout#tag/Session/paths/~1v2~1session/post). The dynamic logistics callback is found by clicking the "Callback" tab in the Swagger.
```
Comment (sebdak) [inline vipps-checkout-api.md:203]: The last sentence of this paragraph is not relevant any more. Maybe reformulate? Or remove?

**Review by sebdak — APPROVED** [review]
Sweet! One minor comment

File: vipps-checkout-api.md (hunk 1 above)
Comment (rebekaburnett) [reply vipps-checkout-api.md]: I removed the last sentence and also changed [Swagger] to [API Reference], which is what we are saying everywhere now.

**Comment by rebekaburnett** [comment]
Note - once I get the epayments into the new website, I will fix all those links, too.

**Pull request merged**